index.js

```diff
@@ -50,6 +50,57 @@ client.on('message', message => {
         .send(MESSAGE_PREFIX + "Send '!verify' to access other channels")
         .catch(reason => console.log(reason))
     }
+    else if (new RegExp(`^${CONFIG.SELF_JOIN_JOIN_COMMAND}|${CONFIG.SELF_JOIN_LEAVE_COMMAND}`).test(text)) {
```

Comment:
```suggestion
    else if (new RegExp(`^${CONFIG.SELF_JOIN_JOIN_COMMAND}`).test(text)) {
```

and can the leave stuff be a separate, top-level condition?

The amount nested if conditions makes this difficult to work with :(

Reply:
can we**

amount of**